### ISSUE TITLE
HostFlagID is inconsistent

### ISSUE THREAD

**Issue by cjcarlson** [viralemergence/virion#48]
```
> vir %>% select(HostFlagID) %>% distinct %>% pull(HostFlagID)
[1]    NA FALSE  TRUE
```

**Comment by cjcarlson** [comment]
Similar bad thing here:
```
> vir %>% select(VirusFlagContaminant) %>% distinct %>% pull(VirusFlagContaminant)
[1]    NA FALSE  TRUE
```

**Comment by cjcarlson** [comment]
Temporary fix in 8654517c75e44b25e573cb65aef76d01f42b180a but could probably benefit from upstream solutions and recompilation

**Comment by cjcarlson** [comment]
Resolved in 88d3692d3cdc17364ea06b29114b8eae420db934
